environment.go

```diff
@@ -89,6 +98,14 @@ func (e *envState) Configure(ctx context.Context, confPath string) error {
 	catcher.Add(e.createQueues(ctx))
 	catcher.Extend(e.initQueues(ctx))
 
+	if !catcher.HasErrors() {
+		var err error
+		e.clientConfig, err = getClientConfig(e.settings.Ui.Url)
+		catcher.Add(err)
+		if err == nil && len(e.clientConfig.ClientBinaries) == 0 {
+			grip.Warning("No clients are available for this server")
+		}
+	}
```

Comment:
I'd write a (initClientConfig) method that returns an error, and does the warning, and just call catcher.Add() here.

Reply:
done